.github/workflows/check-catalyst.yaml

```diff
@@ -317,7 +374,7 @@ jobs:
       uses: actions/cache@v3
       with:
         path: mlir/llvm-project
-        key: Linux-llvm-${{ steps.llvm-hash.outputs.llvm-hash }}-default-source
+        key: Linux-llvm-${{ steps.llvm-hash.outputs.llvm-hash }}-sanitize-source
```

Comment:
There's no need for a special source cache since they will be identical :)

Reply:
:smile: